provider/qemu_img_utils.py

```diff
@@ -115,3 +115,32 @@ def check_flag(strace_log, target_file, flag):
         for line in lines:
             logging.debug(line.strip())
         return any(flag in line for line in lines)
+
+
+def setup_loop_dev(export_img):
```

Comment:
this could simply be like:
```python
def setup_loop_dev(image_path):
    cmd_result = process.run("losetup -f %s --show" % image_path)
    return cmd_result.stdout_text.strip()
```